composer.json

```diff
@@ -20,7 +20,8 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.3",
+		"php": ">=5.3.9",
```

Comment:
The needed symfony components are in Symfony 2.8, which has a min. php requirement of 5.3.9.

Since this is a dev tool for offline use by ext developers, it shouldn't be an issue.